packages/ui/src/mixins.js

```diff
@@ -46,6 +46,13 @@ export const typography = (level) => {
       color: ${colors.F2};
       font-weight: 400;
     `,
+
+    p3: css`
+      font-size: 12px;
+      line-height: 22px;
+      color: rgba(0, 0, 0, .65);
```

Comment:
The color is not from our existing palette. Please update with a similar color in the palette.